content/docs/cucumber/state.md

```diff
@@ -476,12 +529,14 @@ With Rails, you can also turn off transaction globally in your `features/support
 ```
 Cucumber::Rails::World.use_transactional_fixtures = false
 ```
+{{% /block %}}
 
-## Cleaning Your Database
+{{% block "java,kotlin,javascript" %}} Ruby tools provide specific ways to turn of transactions. {{% /block %}}
```

Comment:
I'm puzzled why there's something about Ruby for the Java, kotlin, JS tabs?

Reply:
See above